Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM 289208114389.dkr.ecr.us-east-1.amazonaws.com/picpay-dev/hyperf:php8.2-swoole5.0
+
+ARG COMPOSER_AUTH
+ENV COMPOSER_AUTH $COMPOSER_AUTH
+COPY composer.* .
+RUN composer install --prefer-dist --no-dev --optimize-autoloader
+
+COPY . .
+ENTRYPOINT [ "sh" ]
+
+ARG APP_ENV="prod"
+ENV APP_ENV $APP_ENV
+RUN if [ "$APP_ENV" = "dev" ]; then \
```

Comment:
I think the whole `Dockerfile` will be used for development environment. Do we really need this condition?